packages/react/src/components/accordion/accordion.tsx

```diff
@@ -19,6 +19,7 @@ interface AccordionProps {
     id?: string;
     items: ItemsProps[];
     mode?: 'single' | 'multi';
+    handleToggle?: (uniqueId: string, expanded: boolean) => void;
```

Comment:
Je suis pas certain, mais peut-être renommer pour matcher le naming dans le component.
```suggestion
    handleToggle?: (itemId: string, expanded: boolean) => void;
```